File: Majo/Act-Instructor_Henry/lenguajes/ruby/mayo/27-05-24.rb
puts "Número de declaración de renta"
m = gets.chomp.to_f

if m <= 10
    renta1 = (m * (20/100.to_f))
    renta = m - renta1
    puts("El valor final es: #{renta}$")
elsif m <= 100
    renta1 = (m * (10/100.to_f)) 
    renta = m - renta1
    puts("El valor final es: #{renta}$")
elsif m <= 1000
    renta1 = (m * (5/100.to_f)) 
    renta = m - renta1
    puts("El valor final es: #{renta}$")
else 
    renta1 = (m * (1/100.to_f))
    renta = m - renta1
    puts("El valor final es: #{renta}$")
end

File: Majo/Act-Instructor_Henry/lenguajes/dart/mayo/27-05-24.dart
import 'dart:io';

void main(){
  print('Número de declaración de renta');
  int m = int.parse(stdin.readLineSync()!);

  if (m <= 10){
    double renta1 = (m * 20/100);
    double renta = m - renta1;
    print('El valor final es: $renta\$');
  }else if (m <= 100){
    double renta1 = (m * 10/100); 
    double renta = m - renta1;
    print('El valor final es: $renta\$');
  }else if (m <= 1000){
    double renta1 = (m * 5/100);
    double renta = m - renta1; 
    print('El valor final es: $renta\$');
  }else {
    double renta1 = (m * 1/100); 
    double renta = m - renta1;
    print('El valor final es: $renta\$');
  }
}